src/Linter/Yaml/YamlLinter.php

```diff
@@ -78,6 +79,19 @@ public static function supportsFlags()
         return $params[1]->getName() === 'flags';
     }
 
+    /**
+     * This method can be used to determine the Symfony Linter version.
+     * If this method returns true, you are using Symfony YAML >= 4.0.0.
+     *
+     * @link http://symfony.com/blog/new-in-symfony-3-1-yaml-deprecations#deprecated-the-dumper-setindentation-method
```

Comment:
looks like a great candidate to add `@todo drop me after ...` as well ;)

Reply:
Not sure. Guess the `supportsFlags` method above should receive the same treatment, then?

Reply:
indeed, it shall.